app/src/main/java/com/kahuis/repairtool/data/remote/RepairApi.kt

```diff
@@ -0,0 +1,22 @@
+package com.kahuis.repairtool.data.remote
+
+import com.kahuis.repairtool.data.remote.dto.RepairDetailDto
+import com.kahuis.repairtool.data.remote.dto.RepairDto
+import retrofit2.http.GET
+import retrofit2.http.POST
+import retrofit2.http.Path
+
+interface RepairApi {
+
+    //Get all repairs available
+    @GET("/get_repairs/")
+    suspend fun getRepairs(): List<RepairDto>
```

Comment:
I think there is some inconsistency between your abstraction of the resources and the resources that the REST API will use. Here you're for example calling a `ReparationRequest` a `Repair`. Although it's definitely possible to create your own abstraction layer on top of the API defined by the main application, you will probably always make an abstraction first for the API. Right now, new developers that will contribute to this project will have trouble making a connection between the documentation from the REST API and your abstraction. 

Something to think about 👍